backend/routers/social_media_marketing_router.py

```
@@ -828,57 +828,104 @@ async def get_swamiji_avatar_config(request: Request, admin_user: dict = Depends
         ) from e
 
 
-@social_marketing_router.post("/generate-avatar-preview")
+@social_marketing_router.post("/generate-avatar-preview", response_model=StandardResponse, tags=["Social Media Marketing", "Admin"])
 async def generate_avatar_preview(
-    request: AvatarPreviewRequest,
-    admin_user: dict = Depends(get_admin_user)
+    request: GenerateAvatarPreviewRequest,
```

Ruff (0.12.2)

Undefined name `GenerateAvatarPreviewRequest`

(F821)